Update buienradar.markdown for config flow (#14000)
home-assistant/home-assistant.github.io: source/_integrations/buienradar.markdown

  - Weather
ha_release: 0.47
ha_iot_class: Cloud Polling
ha_config_flow: true
ha_codeowners:
  - '@mjj4791'
  - '@ties'
  - '@Robbie1221'
ha_domain: buienradar
ha_platforms:
  - camera
  - sensor
  - weather
---

> The `buienradar` integration uses [buienradar.nl](https://buienradar.nl/) as a source for current meteorological data for your location. The weather forecast is delivered by Buienradar, who provides a web service that provides detailed weather information for users in The Netherlands.

Reviewer: ```suggestion
The Buienradar integration uses [buienradar.nl](https://buienradar.nl/) as a source for current meteorological data for your location. The weather forecast is delivered by Buienradar, who provides a web service that provides detailed weather information for users in The Netherlands.
```

Besides the weather platform, there is currently support for the following additional device types:

- [Camera](#camera): Radar map
- [Sensor](#sensor): Extended weather data

{% include integrations/config_flow.md %}

## Camera

The `buienradar` camera platform uses [buienradar.nl](https://buienradar.nl/) as a source for the last rain radar map. The overview image of the whole of the Netherlands is loaded and shown as a camera in Home Assistant. The Netherlands is the default country and can be changed to Belgium (see [Setting options](#setting-options)).

Internally this component uses the radar map image as [documented](https://www.buienradar.nl/overbuienradar/gratis-weerdata) on buienradar.nl.
The downloaded image is cached to prevent Home Assistant from making a new request to buienradar.nl multiple times a minute when Home Assistant checks for new stills from the camera.

The camera entity is added disabled by default and should first be enabled before it starts reading the camera images.

## Sensor

> The `buienradar` sensor platform will setup separate sensor entities with more detailed weather data. The selected weather station will provide all weather data, with the exception of the forecasted precipitation. The forecasted precipitation data will be retrieved from buienradar using your actual gps-location (and not the location of the nearest weather station). The sensor entities are disabled by default and should be enabled before they will be updated with data.

Reviewer: ```suggestion
The Buienradar integration will set up separate sensor entities with more detailed weather data. The selected weather station will provide all-weather data, with the exception of the forecasted precipitation. The forecasted precipitation data will be retrieved from Buienradar using your actual GPS location (and not the location of the nearest weather station). The sensor entities are disabled by default and should be enabled before they will be updated with data.
```

The following entities will be created:

> - `stationname`: The name of the selected meteo-station.

Reviewer: We use code fencing for code & file names.
This might make people think this is the name of the entity (which is not guaranteed and can be customized).

Instead, just make a list of descriptions, or, replace the code fence with `**Station name**`

- `winddirection_1d` `winddirection_2d` `winddirection_3d` `winddirection_4d` `winddirection_5d`: "Where the wind will be coming from: N (North), Z (south), NO (North-East), etc."
- `windforce_1d` `windforce_2d` `windforce_3d` `windforce_4d` `windforce_5d`: "The expected windforce in [Bft](https://en.wikipedia.org/wiki/Beaufort_scale)."
- `windspeed_1d` `windspeed_2d` `windspeed_3d` `windspeed_4d` `windspeed_5d`: "The expected windspeed in [m/s](https://en.wikipedia.org/wiki/M/s) (derived from windforce_1d)."
- `conditioncode_1d` `conditioncode_2d` `conditioncode_3d` `conditioncode_4d` `conditioncode_5d`: Symbol and condition code of the expected condition.
- `condition_1d` `condition_2d` `condition_3d` `condition_4d` `condition_5d`: Symbol and expected condition.
- `conditiondetailed_1d` `conditiondetailed_2d` `conditiondetailed_3d` `conditiondetailed_4d` `conditiondetailed_5d`: Symbol and detailed expected condition.
- `conditionexact_1d` `conditionexact_2d` `conditionexact_3d` `conditionexact_4d` `conditionexact_5d`: Symbol and full expected condition (in English).
- `symbol_1d` `symbol_2d` `symbol_3d` `symbol_4d` `symbol_5d`: Symbol and full expected condition (in Dutch).

> ## Setting options
> 
> To configure options for `buienradar` integration go to **Configuration** >> **Integrations** and press **Options** on the Buienradar card.

Reviewer: ```suggestion
{% include integrations/option_flow.md %}
```

> {% configuration %}

Reviewer: Use `configuration_basic` instead, that was we can remove all `required`, `default` and `type` parameters (as they are handled and shown in the UI).

  type: string
{% endconfiguration %}

## Sensor

The `buienradar` sensor platform will setup separate sensor entities with more detailed weather data. The selected weather station will provide all weather data, with the exception of the forecasted precipitation. The forecasted precipitation data will be retrieved from buienradar using your actual gps-location (and not the location of the nearest weather station).

> When camera platform is selected, a window will appear for configuration of additional parameters.

Reviewer 2: Shouldn’t “camera” be “sensor” here?
```suggestion
When sensor platform is selected, a window will appear for configuration of additional parameters.
```

Author: This PR needs adjustment, it doesn't match current implementation. I'll markt it draft.
